Classes/Domain/Timestamp.php

```diff
@@ -1,32 +0,0 @@
-<?php
-namespace Neos\Cqrs\Domain;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-/**
- * Timestamp
- */
-class Timestamp
```

Comment:
`Utility\Now` doesn't create the timestamp with microseconds though, so it's not possible to retain them later. We should provide some way to create highly precise timestamps, even if we might not use them ourself in the base implementation.